plugin/templates/_README.md

```diff
@@ -18,23 +18,23 @@ npm install eslint-plugin-<%= pluginId %> --save-dev
 
 ## Usage
 
-Add `<%= pluginId %>` to the plugins section of your `.eslintrc` configuration file. You can omit the `eslint-plugin-` prefix:
+Add `<%= pluginId %>` to the `plugins` key of your [configuration file](https://eslint.org/docs/latest/use/configure/configuration-files#configuration-file). You can omit the `eslint-plugin-` prefix:
 
-```json
+```js
 {
-    "plugins": [
+    plugins: {
         "<%= pluginId %>"
```

Comment:
Adding a plugin name as a string to `plugins` won't work in the new config.

We could do something like this:

```js
import <%= pluginId %> from "eslint-plugin-<%= pluginId %>";

export default [
    {
        plugins: {
            <%= pluginId %>
        }
    }
];
```

and below in the rules section:

```js
import <%= pluginId %> from "eslint-plugin-<%= pluginId %>";

export default [
    {
        plugins: {
            <%= pluginId %>
        },
        rules: {
            "<%= pluginId %>/rule-name": "warn"
        }
    }
];
```

Reply:
Thanks for the suggestion!
Resolved in https://github.com/eslint/generator-eslint/pull/194/commits/471f2d33d7f786f4667f36bdc4a0479013ba67b3